cli/azd/pkg/auth/manager.go

```diff
@@ -432,14 +440,24 @@ func (m *Manager) LoginWithDeviceCode(
 		return nil, err
 	}
 
-	// Display the message to the end user as to what to do next, then block waiting for them to complete
-	// the flow.
-	fmt.Fprintln(deviceCodeWriter, code.Message())
+	m.console.MessageUxItem(ctx, &ux.MultilineMessage{
+		Lines: []string{
+			fmt.Sprintf("Start by copying the next code: %s", output.WithBold(code.UserCode())),
+			"Then press enter and continue to log in from your browser...",
+		},
+	})
+	m.console.WaitForEnter()
 
+	url := "https://microsoft.com/devicelogin"
+	if err := m.launchBrowserFn(url); err != nil {
+		m.console.Message(ctx, fmt.Sprintf("Error launching browser. Manually go to: %s", url))
```

Comment:
We should log `err` to debug for troubleshooting

Reply:
done